### DISCUSSION TITLE
建议限制默认情况下guest的权限

### DISCUSSION THREAD

**Discussion by wenfer** [AlistGo/alist#3793] [General]
现在可以通过ip扫描默认端口的方式，扫到很多没有配置guest权限的服务，可以直接进去浏览下载他们的资源。
而且配置guest权限的方式不是很直观

**Comment by anwen-anyi** [comment]
现在guest权限是停止使用的 
 请更新你的版本 
如果你是升级的需要手动去停止
如果你是第一次安装默认是停止使用状态
